docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
```

Comment:
With some "set once" fields we can at least merge empty (those with omitempty or a specific swagger flag) on the client side, which makes reload more palatable.


> On Jan 7, 2015, at 7:03 PM, bgrant0607 <notifications@github.com> wrote:
> 
> In docs/cli-usage.md:
> 
> > @@ -0,0 +1,96 @@
> > +## Using Kubernetes from the command line
> Since there are fields that can't be updated (e.g., PortalIP), we probably also need to support reload, to delete and recreate an object with a new configuration.
> 
> We probably should think about creation via update and update via creation (in the case that the object exists but differs).
> 
> —
> Reply to this email directly or view it on GitHub.
>